stable/chartmuseum/README.md

```diff
@@ -282,8 +282,8 @@ helm install --name my-chartmuseum -f custom.yaml stable/chartmuseum
 ```
 
 ### Using with local filesystem storage
-By default chartmuseum use local filesystem storage. 
-But on pod recreation if will lose all charts, to prevent that enable persistent storage. 
+By default chartmuseum uses local filesystem storage. 
+But on pod recreation it will lose all charts, to prevent that enable persistent storage. 
```

Comment:
Would you also fix the second half of this sentence ? It reads funny...

Reply:
Hi, @DStorck Thanks for your attention.
The second half means to prevent losing all charts, the user need to enable persistent storage. Maybe we can fix it to "to prevent that happening, it needs to enable persistent storage"
Do you think that is OK?